gui-wm/river/river-0.2.3.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 2021-2022 Aisha Tammy
+# Distributed under the terms of the ISC License
+
+EAPI=8
+
+DESCRIPTION="Dynamic tiling wayland compositor"
+HOMEPAGE="https://github.com/riverwm/river"
+
+SRC_URI="https://github.com/riverwm/river/releases/download/v${PV}/${P}.tar.gz"
+KEYWORDS="~amd64"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+man pie test +X"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/libevdev
+	dev-libs/libinput
+	dev-libs/wayland
+	>=gui-libs/wlroots-0.16.0:=[X?]
+	x11-libs/libxkbcommon:=[X?]
+	x11-libs/pixman
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	|| ( >=dev-lang/zig-0.10.0 >=dev-lang/zig-bin-0.10.0 )
+	dev-libs/wayland-protocols
+	man? ( app-text/scdoc )
+	virtual/pkgconfig
+"
+
+QA_FLAGS_IGNORED="usr/bin/river(ctl|tile)?"
+
+src_compile() {
+	local zigoptions=(
+		--verbose
+		-Drelease-safe
+	)
+
+	if use man ; then
+		zigoptions+=("-Dman-pages")
+	fi
+
+	if use pie ; then
+		zigoptions+=("-Dpie")
+	fi
+
+	if use X ; then
+		zigoptions+=("-Dxwayland")
+	fi
+
+	zigoptions+=("${EXTRA_ECONF[@]}")
```

Comment:
```suggestion
```
I don't think that should be here as EXTRA_ECONF (if not empty) contains flags for `configure` (thats in the name). Might instead use smth like `ZIG_FLAGS`?

Reply:
I agree with @BratishkaErik, please change them to `ZIG_FLAGS`